operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -166,9 +166,15 @@ limits_config:
   split_queries_by_interval: 30m
 memberlist:
   abort_if_cluster_join_fails: true
-  bind_port: {{ .GossipRing.Port }}
+  {{- with .GossipRing.BindAddrs }}
+  bind_addr:
+    {{- range . }}
+    - {{ . }}
+    {{- end }}
+  {{- end }}
+  bind_port: {{ .GossipRing.BindPort }}
   join_members:
-    - {{ .GossipRing.FQDN }}:{{ .GossipRing.Port }}
+    - {{ .GossipRing.MembersDiscoveryAddr }}:{{ .GossipRing.BindPort }}
```

Comment:
Does it make sense to put the whole "memberlist configuration block" into a `{{ With .GossipRing }}`?